src/System.Net.Requests/ref/System.Net.Requests.cs

```diff
@@ -355,93 +407,47 @@ public abstract partial class WebResponse : System.MarshalByRefObject, System.Ru
         public virtual void Close() { }
         public void Dispose() { }
         protected virtual void Dispose(bool disposing) { }
-        public virtual System.IO.Stream GetResponseStream() { throw null; }
         protected virtual void GetObjectData(System.Runtime.Serialization.SerializationInfo serializationInfo, System.Runtime.Serialization.StreamingContext streamingContext) { }
+        public virtual System.IO.Stream GetResponseStream() { throw null; }
         void System.Runtime.Serialization.ISerializable.GetObjectData(System.Runtime.Serialization.SerializationInfo serializationInfo, System.Runtime.Serialization.StreamingContext streamingContext) { }
     }
-    [ObsoleteAttribute("This class has been deprecated. Please use WebRequest.DefaultWebProxy instead to access and set the global default proxy. Use 'null' instead of GetEmptyWebProxy. http://go.microsoft.com/fwlink/?linkid=14202")]
-    public class GlobalProxySelection
-    {
-        public static IWebProxy Select { get { throw null; } set { } }
-        public static IWebProxy GetEmptyWebProxy() { throw null; }
-    }
-    public enum FtpStatusCode
-    {
```

Comment:
It's moved here :https://github.com/dotnet/corefx/pull/13874/files/6dc2163b366cea03f80099b3bf41680f32ea408a#diff-ce48c170cda4f1a381cd880e4ca56562R73

The reason is that Standard generated system.cs automatically and having the same ordering simplifies code diffing. These files seem to have been generated using a different tool or manually.